specification/app/Microsoft.App.DynamicSessions/routes.tsp

```diff
@@ -92,3 +94,35 @@ interface SessionResourceFiles {
     }
   >;
 }
+
+@route("/session")
+interface SessionManagement {
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "No suitable standard operation found."
+  @doc("Get the session.")
+  @get
+  get is Foundations.Operation<SessionIdentifier, Session>;
+
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "No suitable standard operation found."
```

Comment:
Can't we avoid these suppression as it means they are not following standard pattern ?

Reply:
Yeah I tried to use `resource` Operations which is the standard, but that requires resource Id i.e sessionIdentifier added in the URL Path instead of query params, which is not what we follow for other data-plane APIs. So these suppressions are required.